src/strings.ts

```diff
@@ -213,6 +213,7 @@ export const catalog: Dict = {
         props: {
           dateCreated: () => 'Date created',
           lastSync: () => 'Last sync',
+          namespace: () => 'Namespace',
```

Comment:
I forgot that we call a namespace a "project" in MCC. Let's use `'Project'` instead of `'Namespace'`

```suggestion
          namespace: () => 'Project',
```

Reply:
Fixed